doc/doc/advanced/deepdiveapp.md

```diff
@@ -37,10 +43,11 @@ A DeepDive application is a directory that contains the following files and dire
 
 * `run/`
 
-    Each run of the DeepDive application has a corresponding subdirectory under this directory whose name contains the timestamp when the run was started, e.g., `run/20150618-223344.567890/`.
+    Each run of the DeepDive application has a corresponding subdirectory under this directory whose name contains the timestamp when the run was started, e.g., `run/20150618/223344.567890/`.
     All output and log files that belong to the run are kept under that subdirectory.
     There are a few symbolic links with mnemonic names to the most recently started run, last successful run, last failed run for handy access.
 
+[DDlog]: ../basics/ddlog.html
```

Comment:
Well, I also wish we could use `.md`, but Jekyll, our website compiler doesn't convert that automatically into `.html`.